### ISSUE TITLE
discrepancies between VSPK and VSD api specs

### ISSUE THREAD

**Issue by hellt** [nuagenetworks/vsd-api-documentation#5]
Hi @pdellaert 
looking at [connectionendpoint](https://nuagenetworks.github.io/vsd-api-documentation/v5_0/connectionendpoint.html) object I see that the required field of vsd-api-specs is altered in [vspk doc article](https://nuagenetworks.github.io/vspkdoc/v5_0/nuconnectionendpoint.html).

In VSPK the ip_address is marked as mandatory, whereas its not required in the specs. How do we generate the vspk docs?

**Comment by pdellaert** [comment]
yeah, the vspk_doc is outdated at this point, and i need to fix that... it's something on my todo
